### DISCUSSION TITLE
Docker Install Notes

### DISCUSSION THREAD

**Discussion by disruptivepatternmaterial** [Netatalk/netatalk#756] [General]
Thank you for the people who did this, it is so helpful for turnkey semi disposable setups like the ones I mostly work with...

A couple notes that I think should be in the README to help make it easier:

The image in the README failed for me, had to make it` image: netatalk/netatalk3:latest`

and  a volume of `- /home/me/netatalk/afp.conf:/usr/etc/afp.conf` allowed me to control the setup easier with a custom conf.

you should put in all the group/user vars in the compose as normal, I also put in `MANUAL_CONFIG=yes` but uncertain if that is even needed if you mount over the conf file as I did, but I did just in case.

**Comment by rdmark** [comment]
Thanks for sharing your feedback. I’m pleased to hear that you found utility in the docker image. 

With regards to the README on GitHub, when I wrote that, the intention was for users to build their own images and then launch them locally. If you compare to the README on Docker Hub you can see the difference. However I can revisit the wording in the former and see if I can prevent future misunderstandings.

**Reply by disruptivepatternmaterial** [reply]
Ah I didnt even read the README on Docker Hub, assuming the one here was the golden version. Thank for the reply!
